R/ridge.R

```diff
@@ -84,9 +83,19 @@ fit_ridgeaug_formatted <- function(wide_data, synth_data,
         if(ridge) {
             if(is.null(lambda)) {
                 if(ncol(y) > 1) {
```

Comment:
I don't think we need this check anymore if we're changing the lambdas

Reply:
Should we overwrite the lambda if it is non-null?

Reply:
Ah I was referring to `if(ncol(y) > 1)`. We just do the same thing regardless now, that was a check that mattered for `glmnet`